docs/installation.rst

```diff
@@ -22,8 +22,8 @@ Python 3.7+ and pysat 3.0.0+.
  ================ ==================
   numpy            aacgmv2
   pandas           apexpy
-  pyEphem          OMMBV
-  sgp4             pysat>=3.0
+  pyEphem          OMMBV>=1.0
+  sgp4>=2.7        pysat>=3.0
  ================ ==================
 
 
```

Comment:
This is intended for the end of this file.
```

.. _post-install:
Post Installation
-----------------

After installation, you may register the :py:mod:`pysatMissions`
:py:class:`Instrument` sub-modules with pysat.  If this is your first time using
pysat, check out the `quickstart guide
<https://pysat.readthedocs.io/en/latest/quickstart.html>`_ for pysat. Once pysat
is set up, you may choose to register the the :py:mod:`pysatMissions`
:py:class:`Instruments` sub-modules by:

.. code:: python


   import pysat
   import pysatMissions

   pysat.utils.registry.register_by_module(pysatMissions.instruments)

You may then use the pysat :py:attr:`platform` and :py:attr:`name` keywords to
initialize the model :py:class:`Instrument` instead of the
:py:attr:`inst_module` keyword argument.
```